apps/opencs/model/world/columnimp.hpp

```diff
@@ -2265,6 +2264,148 @@ namespace CSMWorld
             return true;
         }
     };
+
+    template<typename ESXRecordT>
+    struct PathgridPointListColumn : public Column<ESXRecordT>
+    {
+        PathgridPointListColumn ()
+        : Column<ESXRecordT> (Columns::ColumnId_PathgridPoints,
+                ColumnBase::Display_PathgridPointList, ColumnBase::Flag_Dialogue)
+        {
+        }
+
+        virtual QVariant get (const Record<ESXRecordT>& record) const
+        {
+            return true; // required by IdTree::hasChildren()
+        }
+
+        virtual void set (Record<ESXRecordT>& record, const QVariant& data)
+        {
+        }
+
+        virtual bool isEditable() const
+        {
+            return true;
+        }
+    };
+
+    template<typename ESXRecordT>
+    struct PathgridIndexColumn : public Column<ESXRecordT>
+    {
+        PathgridIndexColumn()
+        : Column<ESXRecordT> (Columns::ColumnId_PathgridIndex, ColumnBase::Display_Integer)
+        {}
+
+        virtual QVariant get (const Record<ESXRecordT>& record) const
+        {
+            return QVariant(); // FIXME
```

Comment:
Actually, they are intended to return invalid values. Since these columns have sub-rows, the methods in idadapterimp.hpp are used instead.